src/views/FormCustomerView.vue

```diff
@@ -11,29 +11,25 @@
         <input v-model="vorname" type="text" id="vorname" required />
       </div>
       <div class="form-group">
-        <label for="kundenNr">Kundennr.:</label>
-        <input v-model="kundenNr" type="text" id="kundenNr" />
+        <label for="strasse">Straße:</label>
+        <input v-model="strasse" type="text" id="strasse" />
       </div>
       <div class="form-group">
-        <label for="adresse">Adresse:</label>
-        <input v-model="adresse" type="text" id="adresse" />
+        <label for="adresse">Plz:</label>
```

Comment:
bitte ändere zeile in : <label for="plz">Plz:</label>

Reply:
geändert.